fixtures/themed/src/styles.css.ts

```diff
@@ -67,7 +67,6 @@ globalStyle(`body ${iDunno}:after`, {
 const blankVar1 = createVar({
   syntax: '<number>',
   inherits: false,
-  initialValue: '0',
```

Comment:
removed initial value to make fallback function `fallbackVar(blankVar1, '0.5')` work